rel/overlay/etc/default.ini

```diff
@@ -598,3 +598,14 @@ min_priority = 2.0
 ; log every generated trace by including the following:
 ;
 ; all = (#{}) -> true
+
+[expiring_cache]
+;
+; How often to remove expired entries
+; period_msec = 5000
+;
+; Jitter applied when removing expired entries
+; max_jitter_msec = 1000
+;
+; Maximum number of entries removed in a single transaction
+; batch_size = 1000
```

Comment:
Since it's a library, let's skip updating the default.ini changes. Each application can then have its own section

Reply:
Argh, I forgot to remove this when I converted to a library. Done.